src/main/kotlin/com/autonomousapps/subplugin/RootPlugin.kt

```diff
@@ -18,6 +18,8 @@ import org.gradle.api.artifacts.Configuration
 import org.gradle.kotlin.dsl.apply
 import org.gradle.kotlin.dsl.register
 
+const val DEPENDENCY_ANALYSIS_PLUGIN = "com.autonomousapps.dependency-analysis"
```

Comment:
Do we want this public, or is `internal` better?

Reply:
Yes `internal` is good. Thanks.

It's not in my repertoire as "doing a lot of Java" programmer. I tend to forget about it. 😅